storybook/storybook-react/src/Button/Button.docs.mdx

```diff
@@ -1,15 +1,11 @@
-import { Canvas, Controls, Markdown, Meta, Primary } from "@storybook/blocks";
+import { Canvas, Markdown, Meta } from "@storybook/blocks";
 import * as ButtonStories from "./Button.stories.tsx";
 import README from "../../../../packages/css/src/components/button/README.md?raw";
 
 <Meta of={ButtonStories} />
 
 <Markdown>{README}</Markdown>
 
-<Primary />
-
-<Controls />
-
```

Comment:
Why did you remove these?

Reply:
They interupted the docs without added value.